ykfr/src/lib.rs

```diff
@@ -70,7 +69,11 @@ impl Frame {
 
 fn get_stackmap_call(pc: Value) -> Value {
     debug_assert!(pc.is_instruction());
-    let sm = unsafe { Value::new(LLVMGetPreviousInstruction(pc.get())) };
+    let sm = if pc.is_call() {
```

Comment:
One case for branches, one case for calls? SHould we add some comments?

Reply:
Try [55ccf18](https://github.com/ykjit/yk/pull/589/commits/55ccf187578d930b3945d6f6b2e7fde3e9d455c7)

Reply:
perfect.